src/containers/bit-brush-mode.jsx

```diff
@@ -5,11 +5,10 @@ import bindAll from 'lodash.bindall';
 import Modes from '../lib/modes';
 import {MIXED} from '../helper/style-path';
 
-import {changeFillColor, DEFAULT_COLOR} from '../reducers/fill-color';
+import {changeFillColor, clearFillGradient, DEFAULT_COLOR} from '../reducers/fill-swatch';
```

Comment:
Since swatches are going to be a separate thing, we probably want to choose a different word...

Reply:
fill-gradient ??

Reply:
fill-color-or-gradient?

Reply:
Note to self: fill-style and stroke-style